yax/artifacts/informative_alignment_data.py

```diff
@@ -0,0 +1,21 @@
+from yax.state.type.artifact import Artifact
+from yax.shared.utilities import taxidtool
+
+
+class InformativeAlignmentData(Artifact):
+    def __init__(self, completed):
+        self.tax_id_data = None
+        self.inclusion_tree = None
+        self.gis_to_taxids = None
+        self.sequences_input_fp = None
+        self.truncation_level = None
+
+        if completed:
+            pass
```

Comment:
the process of running the module populates these, which are then used in completion to write .seqs and .tree files.

I was not, and am still unsure on how to reproduce these values from the completed artifact... I may have to create a whole other file type to store this stuff as it can not be inferred(reliably?) from .seqs and .tree but I thought that seemed inelegant